jdwp-shellifier.py

```diff
@@ -12,16 +12,18 @@
 import time
 import sys
 import struct
-import urllib
+import urllib.request, urllib.parse, urllib.error
 import argparse
+import traceback
+import pdb
 
 
 
 ################################################################################
 #
 # JDWP protocol variables
 #
-HANDSHAKE                 = "JDWP-Handshake"
+HANDSHAKE                 = b"JDWP-Handshake"
```

Comment:
Can you also edit the README too to mention the script requires Python3 (tested on Python 3.9)?